tests/app/storage/conftest.py

```diff
@@ -17,3 +26,50 @@ def fake_eq_session():
     )
 
     return eq_session
+
+
+@pytest.fixture
+def ddb():
```

Comment:
Might help to not abbreviate, fine here as I can look straight down to see what it is. In the test the fixture isn't as obvious